capa-spi-aws-log/src/main/java/group/rxcloud/capa/spi/aws/log/plugin/AwsLogPlugin.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package group.rxcloud.capa.spi.aws.log.plugin;
+
+import group.rxcloud.capa.component.log.DefaultLogPlugin;
+import group.rxcloud.capa.component.log.LogPlugin;
+import group.rxcloud.capa.spi.aws.log.configuration.LogConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * Aws log plugin.
+ */
+public class AwsLogPlugin implements LogPlugin {
```

Comment:
If we don't use this interface in capa-aws, it's a bit strange to define this interface and impl in capa-aws.

Reply:
I think we can define this plugin in `adptor`.